### PR TITLE
Snackbar

### PR DESCRIPTION
**Pull request by rishabh486**
pr generated for snackbar
Screenshot (32)

### CODE REVIEW

File: snackbar.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+.snack-exit {
+  border-radius: 50%;
+  width: 3rem;
+  height: 3rem;
+  border: none;
+}
+
+.show-bar {
+  visibility: visible !important;
+  /* animation:fadein 0.5s; */
```
Comment (DivyanshSareen) [inline snackbar.css]: remove comment

**Review by DivyanshSareen — APPROVED** [review]
just use var for colors and you are good to merge 👍

**Review by rutu14 — APPROVED** [review]
UI suggestion: The white on blue doesn't seem readable. Can have a different shade.

Use variables for color and it's good to merge.

File: snackbar.css (hunk 1 above)
Comment (rishabh486) [reply snackbar.css]: removed the comment

**Pull request merged**